app/src/modules/settings/routes/flows/components/operation-detail.vue

```diff
@@ -121,7 +121,7 @@ function saveOperation() {
 
 	saving.value = true;
 
-	const defaultValues = operationOptions.value
+	const defaultValues = Array.isArray(operationOptions.value)
```

Comment:
As object (i.e. component) should bypass defaults should we have it return undefined in the `operationOptions` computed on L109? We will no longer need to worry about this caveat for future usage.